documentation/migration-guides/Stack/Azs.Storage.Admin.0.3.0.md

```diff
@@ -0,0 +1,41 @@
+# Table of Contents
+1. [Summary](#summary)
+2. [Breaking changes in storage admin module](#Breaking-changes-in-storage-admin-module)
+
+## Summary
+The module version 0.3.0 of AzureStack SRP brings in many breaking changes. The support version of SRP is upgraded from 2015-12-01-preview to 2019-08-08-preview. Some of the commands and the concept of **FARM** are removed to simplify the usage of SRP. The metrics is also removed from SRP to MDM.
+
+## Breaking changes in storage admin module
+
+**Deprecated Cmdlets**
+
+- ```Get-AzsStorageFarmMetric```, ```Get-AzsStorageFarmMetricDefinition```, ```Get-AzsQueueServiceMetric```, ```Get-AzsQueueServiceMetricDefinition```, ```Get-AzsBlobServiceMetric```,  ```Get-AzsBlobServiceMetricDefinition```, ```Get-AzsTableServiceMetric```, ```Get-AzsTableServiceMetricDefinition```, ```Get-AzsStorageShareMetric```, ```Get-AzsStorageShareMetricDefinition``` are deprecated. All the metrics are moved to MDM.
+
```

Comment:
Could you share the guidance of how to look up the metrics using MDM powershell?